### PR TITLE
dec comm call announcement

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

**Comment by stefaniebutland** [comment]
@maelle Thank you again for a way more functional comm call page!

A couple of things
1. I'm not able to use markdown to format the event summary text. [Preview](https://deploy-preview-83--ropensci.netlify.app/commcalls/dec2020-datarepos/) shows no bullets and lack of spacing. I decided not to use headings, but when I did, there was no spacing before or after them
2. I'm requesting an edit to the default text in the zoom dropdown, but first will see if I can do it myself :-)

**Comment by maelle** [comment]
@stefaniebutland

* I added the content class to both the summary and resources so now they are styled like content of blog posts (but would not get headings anchors)
* I put your changes in this PR.

**Comment by maelle** [comment]
Comm call in https://deploy-preview-83--ropensci.netlify.app/rbloggers/index.xml

Preview it via https://codebeautify.org/rssviewer, some pictures look huge but apart from that it's fine and overall good enough?

**Comment by maelle** [comment]
(fixed the image size in the RSS feed so now I think things should be fine over at R-bloggers)

File: content/commcalls/2020-12-16/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    img: daniella-lowenberg.jpg
+  - name: Matt Jones
+    bio: Matt Jones is Director of Informatics R&D at the National Center for Ecological Analysis and Synthesis ([NCEAS](https://www.nceas.ucsb.edu/)), Principal Investigator of the [NSF Arctic Data Center](https://arcticdata.io/), and Director of [DataONE](https://www.dataone.org/) at University of California Santa Barbara. Matt on [GitHub](https://github.com/mbjones), [Twitter](https://twitter.com/metamattj), [Website](http://matt.magisa.org/)
+    img: /img/team/matt-jones.jpg
+  - name: Carl Boettiger
+    bio: Carl Boettiger is Assistant Professor in the [Department of Environmental Science, Policy and Management at UC Berkeley](http://ourenvironment.berkeley.edu/), a co-founder and leadership team member of rOpenSci. Carl on [GitHub](https://github.com/cboettig), [Twitter](https://twitter.com/cboettig), [Website](https://www.carlboettiger.info/)
```
Comment (stefaniebutland) [inline content/commcalls/2020-12-16/index.md:49]:
```suggestion
    bio: Carl Boettiger is Assistant Professor in the [Department of Environmental Science, Policy and Management at UC Berkeley](http://ourenvironment.berkeley.edu/), a Co-founder and strategic advisor of rOpenSci. Carl on [GitHub](https://github.com/cboettig), [Twitter](https://twitter.com/cboettig), [Website](https://www.carlboettiger.info/)
```

File: content/commcalls/2020-12-16/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    details: arXiv preprint by Nicholas Tierney, Karthik Ram
+  - url: https://doi.org/10.1177/2053951719836258
+    name: Enforcing public data archiving policies in academic publishing. A study of ecology journals
+    details: Sholler D, Ram K, Boettiger C, Katz DS (2019) Big Data & Society.
+  - url: https://doi.org/10.1371/journal.pone.0230416
+    name: The citation advantage of linking publications to research data
+    details: Colavizza G, Hrynaszkiewicz I, Staden I, Whitaker K, McGillivray B (2020) PLoS ONE 15(4), e0230416. 
+speakers:
+  - name: Kara Woo
+    bio: Kara Woo is a Principal Bioinformatics Engineer at [Sage Bionetworks](http://sagebionetworks.org/) where she leads a team of developers building tools and infrastructure for open science. She has previously worked as an information manager at Washington State University and at the National Center for Ecological Analysis and Synthesis (NCEAS), where she combined data management with fieldwork at a remote Siberian lake. Kara is a member of the rOpenSci Code of Conduct committee. Kara on [GitHub](https://github.com/karawoo), [Twitter](https://twitter.com/kara_woo), [Website](https://karawoo.com/)
```
Comment (stefaniebutland) [inline content/commcalls/2020-12-16/index.md:40]:
```suggestion
    bio: Kara Woo is a Principal Bioinformatics Engineer at [Sage Bionetworks](http://sagebionetworks.org/) where she leads a team of developers building tools and infrastructure for open science. Kara is a member of the rOpenSci Code of Conduct committee. Kara on [GitHub](https://github.com/karawoo), [Twitter](https://twitter.com/kara_woo), [Website](https://karawoo.com/)
```

File: content/commcalls/2020-12-16/index.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    img: kara-woo.jpg
+  - name: Daniella Lowenberg
+    bio: Daniella Lowenberg is the Product Manager at [Dryad](https://datadryad.org/) and Principal Investigator of [Make Data Count](https://makedatacount.org/) within the California Digital Library at University of California. “she was a Publications Manager at PLOS ONE where she implemented and oversaw the PLOS Data Policy. For the last three years, Daniella has been a data publishing and data metrics product manager.” Daniella on [GitHub](https://github.com/dlowenberg), [Twitter](https://twitter.com/danilowenberg)
```
Comment (stefaniebutland) [inline content/commcalls/2020-12-16/index.md:43]:
```suggestion
    bio: Daniella Lowenberg is the Product Manager at [Dryad](https://datadryad.org/) and Principal Investigator of [Make Data Count](https://makedatacount.org/) within the California Digital Library at University of California. Daniella on [GitHub](https://github.com/dlowenberg), [Twitter](https://twitter.com/danilowenberg)
```

**Pull request merged**